### DISCUSSION TITLE
Sorteo de Certificaciones de GitHub - Comparte tu experiencia y gana

### DISCUSSION THREAD

**Discussion by juanpflores** [community/community#118462] [GitHub Education]
### Hi everyone,

¡Bienvenidos al Sorteo de Certificaciones de GitHub! ¿Quieres obtener una certificación oficial de GitHub de forma gratuita? ¡Esta es tu oportunidad!

**¿Cómo participar?**

- Comparte tu experiencia: Publica una foto que refleje tu experiencia en GitHub, ya sea mientras colaboras en un proyecto, aprendes nuevas habilidades o participas en alguna actividad relacionada con GitHub.

- Acompaña con un comentario: Junto a tu foto, comparte un comentario que describa qué te inspira de GitHub, cómo te ha beneficiado o cualquier anécdota interesante que quieras compartir.

**¿Qué puedes ganar?**

Diez afortunados ganadores serán seleccionados al azar para recibir una certificación oficial de GitHub en la especialidad de su elección. ¡Prepárate para mejorar tu perfil profesional con una credencial reconocida por la comunidad de desarrollo!

**Condiciones del sorteo:**

- Debes tener una cuenta de GitHub para participar.
- Cada participante puede ingresar una vez en el sorteo.
- Las fotos y comentarios deben ser apropiados y respetuosos.
- Al participar en el sorteo, das permiso para que tu foto y comentario sean compartidos en las redes sociales de GitHub con el propósito de anunciar a los ganadores y promover futuros eventos similares.
- ¡No pierdas esta oportunidad de mejorar tus habilidades y obtener una certificación de GitHub! Participa ahora y comparte tu pasión por la plataforma. ¡Buena suerte a todos!

**Comment by TheGatoSamurai** [comment]
Desde el comienzo ocupando git hub en la carrera, dándolo todo! 🔥🦾
Lo mejor será siempre innovar tomando lo que nos preste en este tiempo la tecnología y las herramientas. Mis enfoques son respecto a starups, código abierto, y creación de videojuegos. En estos momentos aprendo para darme una línea base, y github a sido un apoyo en Duoc Uc
SALUDOS, de Duoc Uc San Joaquín 

![Screenshot_20240409_161733_YouTube](https://github.com/community/community/assets/102562209/7beb4a7b-7761-4e2f-b429-a6a676128560)
 Ig: kitsune_kotodoma

**Reply by Davbid14** [reply]
![IMG_0509](https://github.com/community/community/assets/166555910/d78e8368-0d5f-4575-bcaa-533b49d5c6cc)

Fue muy interesante lo del problema de los 2 sigma de Bloom, nunca antes lo había escuchado y además, respecto a este, mostraron muy bien un ejemplo de cómo implementar correctamente la inteligencia artificial, al usarla como una forma de aprender mejor en vez de para hacer trampa.

**Comment by RivaulTavara** [comment]
El tema de las IA's es tan interesante, da para horas y horas de discusión 
![Uploading 20240409_155750.jpg…]()

**Comment by fegenau** [comment]
![Captura de pantalla 2024-04-09 162839](https://github.com/community/community/assets/107156361/ea33f503-36c5-4cdb-a265-c06f7b912e03)

La teoria de los dos sigma, me parece interesante. Ahora, es muy caro estar dentro de ese 98%, pero siempre es importante aprender algo nuevo.

**Comment by elvisandrade** [comment]
### 😀**Hace un tiempo me pregunté justamente esto. ¿será que existe una IA que no conozca todo el internet sino algo especifico y enfocarse solo en ese tema?. Bueno, mi pregunta fue respondida y se llama "Educación Personalizada". 🥳🥳 Además, lo del "problema de las dos sigmas" no lo sabía.**😀 ¡¡GRAN CHARLA!!
![IMG_20240409_162306](https://github.com/community/community/assets/62357380/dd9bc3e9-9015-4ab1-bf89-4237801c35af)

**Comment by BenjaminAedo** [comment]
La Arquitectura del futuro
La cual es generar un tutor personalizado para que se encargue de un estudiante siendo un aumento considerable para el alumno ya que crea una ruta personalizada para ver qué temas conocer y ser experto del tema de manera flexible y rapida.

Mejores prácticas para el uso de IA
-Analogía del uso de IA: el uso de IA como esteroide para resolver las dudas de manera rápida aunque tenga un daño a futuro, usarlo como zapatilla pero no tiene daño ni largo o medio plazo, coach la cual ayuda como si fuese un profesor apoyandote 

-Saber hacer preguntas correctas: Orientarnos a resolver el problema y no ser dependientes de está.



![IMG_20240409_162852.jpg](https://github.com/community/community/assets/118937298/3a635057-591c-410e-b161-1f6996e4e7da)

**Comment by ghost** [comment]
![image](https://github.com/community/community/assets/166555671/6ab0e5a3-e241-46d3-8fbf-5b194bbe5373)
Realmente me interesó y me gustó mucho el tema de Copilot, probablemente sea algo que utilizaré en el futuro para poder aprender a hacer código y que me ayude a instruirme de mejor manera dentro de la programación.

**Comment by BenjaminAedo** [comment]
La Arquitectura del futuro
La cual es generar un tutor personalizado para que se encargue de un estudiante siendo un aumento considerable para el alumno ya que crea una ruta personalizada para ver qué temas conocer y ser experto del tema de manera flexible y rapida.

Mejores prácticas para el uso de IA
-Analogía del uso de IA: el uso de IA como esteroide para resolver las dudas de manera rápida aunque tenga un daño a futuro, usarlo como zapatilla pero no tiene daño ni largo o medio plazo, coach la cual ayuda como si fuese un profesor apoyandote 

-Saber hacer preguntas correctas: Orientarnos a resolver el problema y no ser dependientes de está.



![IMG_20240409_162852.jpg](https://github.com/community/community/assets/118937298/3a635057-591c-410e-b161-1f6996e4e7da)

**Comment by frmarch** [comment]
Me pareció bien interesante de cómo podemos integrar la Inteligencia Artificial con el estudio, desconocía la manera correcta de cómo podría sacar provecho a alguna IA para potenciar mis conocimientos, me quedo con que la IA debe ser mi coach en el aprendizaje.

![image](https://github.com/community/community/assets/107442840/874a1aa1-d3cc-40b0-bf4f-c053bbc44a77)

**Comment by Antena19** [comment]
GitHub ha sido una fuente de crecimiento muy importante en el proceso de desarrollo de software durante mis estudios. 
Atenta a la Charla =)
 
 ![Imagen de WhatsApp 2024-04-09 a las 16 40 59_2ca4b180](https://github.com/community/community/assets/131885246/8463ca6e-722c-405e-9d67-67f5d7c0a454)

**Comment by fsebvstian99** [comment]
![Captura de pantalla 2024-04-09 164410](https://github.com/community/community/assets/164553511/495a4af4-99fa-4a0a-93ad-7120f7260799)

Me parece super interesante el tema de codedex y la tutoría personalizada, es algo que empezare a usar, al igual que copilot para potenciar mi aprendizaje y no para que haga mi trabajo, siempre he pensado que la IA, vista como un complemento, una herramienta, es de las mejores cosas que pueden haberse inventado.

**Comment by Camaron99** [comment]
![IMG_6258](https://github.com/community/community/assets/101735499/bd09b48a-beb6-41bc-b481-7c9397300fbf)
Muy interesante la charla la cual habla sobre github y como se puede utilizar la inteligencia artificial para poder masificar la educación 1:1

**Comment by Fergchis** [comment]
Aprendí que puedo trabajar en vscode con Github copilot, lo que hará más sencillo mi desarollo, también tomaré en cuenta el modo 
el cual utilizaré a la IA para mí beneficios. Conocí Github gracias a un amigo programador, actualmente trabajamos juntos en el desarrollo de la página "experimento jazz.cl".

Atte Lucas Huerta
![IMG_20240409_161934276_MFNR](https://github.com/community/community/assets/144072168/827bef4a-ba18-4d06-9da4-b513c52c3ae1)

**Comment by keaguirre** [comment]
Hace tiempo me preguntaba sobre los desafíos en la educación con la aparición de las IA's, aprendí mucho de las propuestas y avances que se están generando para usar estas herramientas como apoyo, gran charla!
![20240409_162636.jpg](https://github.com/community/community/assets/101673253/84bfc797-88cc-47de-b2e2-dfe287196603)

**Comment by GioviOlavarria** [comment]
![imagen](https://github.com/community/community/assets/166557940/e67f0b0b-aeef-4a9b-ba82-48afc35b78ca)
Gracias a esta charla dada por Duoc pude visitar más a fondo la plataforma de Github y me creé una cuenta, me interesó mucho Copilot. Había usado Github anteriormente para ver códigos de otros desarrolladores, y también aplicaciones útiles. Muchas gracias por la charla :)

**Comment by Fuiza** [comment]
Muy buena charla, y muy buena información sobre como usar la inteligencia artificial de forma mas eficiente para el aprendizaje, gracias!
![2024-04-09_17-03](https://github.com/community/community/assets/36244041/fe29654d-cb33-4768-807a-5d8979201c79)

**Reply by fmaripangue** [reply]
Muy buenas exposiciones !!

![image](https://github.com/community/community/assets/101286369/de3f0a6c-8054-48f5-be5f-8abd752a9059)

**Comment by niko12k** [comment]
Gracias por la charla muy interasante sobre todo el como la inteligencia artificial puede ayudar mucho en el aprendizaje 
![aaa](https://github.com/community/community/assets/135355457/df346e12-e603-4654-aac3-367dd14892f1)

**Comment by sebastiandmv2** [comment]
![image](https://github.com/community/community/assets/117068026/cb180bce-d9d8-49f6-b800-8bbd35a53cdb)
Excelente Charla, siempre es bueno aprender un poco mas alla, por lo que el saber como ocupar la IA con copilot para aprender nuevos temas con mayor profundidad es increible,

**Comment by Kuzee21** [comment]
![20240409_170956](https://github.com/community/community/assets/166559232/8eccca96-25e0-4de8-8b32-2d772c86aec4)
Encuentro super interesante el tema de ocupar la inteligencia artificial para una educación mucho más personalizada.
Gracias a esta charla, me interesa investigar más sobre el funcionamiento de GitHub y de como aportar al Open Source.

**Comment by PLUXIN-crt** [comment]
Muy buena charla, me ayudó a entender como poder usar la inteligencia artificial no solo para resolver mis problemas, si no como medio de un aprendizaje más eficiente  entre otras cosas. :) 


![Uploading 20240409_170710.jpg…]()
![Uploading 20240409_162531.jpg…]()

**Comment by Micr0owave** [comment]
muy interesante todo, me encanta el tema de la inteligencia artificial y poder usarlo como un profesor, yo lo que mas he querido es poder estudiar y usar la inteligencia artificial como mi propio profesor, no tengo miedo de preguntarle cosas tontas o poder hacer mas cosas, he estado usando chatgpt para estudiar diferentes tipos de codigos pero nunca me convencion, apoyo mucho el trabajo de "copilot" es realmente genial y importante, me encanta la presentacion y un saludo a todos!!!
![Imagen de WhatsApp 2024-04-09 a las 17 05 16_0b951534](https://github.com/community/community/assets/166555911/a793dafe-bddc-4dde-9d88-cbda81ec2a8c)

**Comment by AntitoFafap** [comment]
Pude aprender lo que es un repositorio, nunca e colaborado en código ya que suelo programar y realizar todo de forma autónoma, es una excelente manera de compartir y ayudar a la comunidad!
![Captura de pantalla 2024-04-09 170741](https://github.com/community/community/assets/138411032/934b94ca-4bf5-480d-8469-a3c3f99f474d)

**Comment by DiegoSantiagos** [comment]
La diferencia entre los tipos de uso de las ias y el hecho de que no se debe usar de forma inadecuada al pedirle las respuestas, en cambio pedirle ayuda para que te ayude a llegar a las respuestas por ti mismo pero con unas facilidadew 
[Cargando IMG-20240409-WA0000.jpeg …]()

**Comment by Javitomon1** [comment]
![20240409_160917](https://github.com/community/community/assets/166559450/7646a21c-ba34-481f-a73c-a0dde936f24a)
Vengo de otra sede a ver la charla, y me parece en demasía interesante lo explicado, enseñado y demostrado dentro de la exposición. Github ya me era familiar pero muy de lejos, por temas de hobbie en css, java y html. Por ende, este vistazo me ha hecho entenderlo mucho más y lo agradezco muchísimo. Además del tema de Copilot, que es fascinante. 

Saludos! Y espero las charlas como estas continúen.

**Comment by Martinpavez913** [comment]
![IMG_5753](https://github.com/community/community/assets/166556020/8c420ce5-ae46-4c31-a81a-d47a1840ae85)
Muy interesante! Me parecen excelentes las ejemplificaciones que utilizaron para mostrar la importancia del manejo de IA 😬

**Comment by Benjamincelis23** [comment]
Una charla muy interesante, aunque usaría muchas más palabras para describir esta sensación de "interés". Sin duda alguna es muy genial y lograre tener más referencias y saber más lo que hago al programar en general. Sin duda los mejores a la hora de charlar.  Todos hombres guapos 👀. 
![IMG_20240409_170614](https://github.com/community/community/assets/134525867/16362724-c8c9-477f-8a51-06a253f33c37)

**Comment by Seba2997** [comment]
Me parecio muy interesante la analogia de utilizar la IA como un coach que te enseñe a como desarrollar mejor mas que un esteroide que haga todo el trabajo por ti, muy buena exposicion! Saludos!
![Captura de pantalla](https://github.com/community/community/assets/129460407/70c6fb93-b40c-4fdc-8621-c43088c154f7)

**Comment by Trukeo** [comment]
Para que se hagan una idea de lo relevante que es para mí esta charla y esta certificación.
Yo desde el 2015 que vengo pateando solo un proyecto de impacto social, soy cero TI y en la soledad siempre creí en mi proyecto, pero nunca hubo nadie que me apoyara, luego llegó la IA a mi vida y el proyecto se transformó, lo traté de mejorar en una presentación y lo presenté a StartUp Chile, y fui rechazado (ellos no saben ver un unicornio aunque lo tengan en frente) y decidí levantarlo solo. Por eso me metí a estudiar Ingeniería en infraestructura tecnológica. Y mientras estaba me arreglaba para salir (porque estudio vespertino) veo la notificación de esta charla en el correo, me meto y lo primero que veo es a un man hablando de lo que yo vengo intentando desarrollar solo desde 2015. Si hubiera nacido en EEUU probablemente estaría realizando todas las ideas de impacto social que tengo, pero aquí estamos, solos y esperando crear comunidad. 
Espero poder ganar, pero sobre todo poder abarcar contactos porque hay otro proyecto que urge y necesito apoyo, ya que no lo estoy teniendo. 

![Captura de Pantalla 2024-04-09 a la(s) 17 02 37](https://github.com/community/community/assets/166558890/66034ecd-39de-4d35-9347-b0a56f198328)

**Comment by Blackmist02** [comment]
Una excelente oportunidad para aprender y crecer en el uso de Github y Github copilot, además como utilizar de una manera eficiente la manera eficiente la IA y lograr mejorar la eficiencia, tanto al momento de aprender y al momento de programar, gracias por dar charlas como esta, ojala se vuelva a repetir una oportunidad como esta.

![Imagen de WhatsApp 2024-04-09 a las 17 06 47_a4f55dbe](https://github.com/community/community/assets/134546421/aa79941b-bad5-4bbe-ab0e-9637f6645339)

**Comment by jhonnylarry** [comment]
Me parece súper interesante como las IA las podemos implementar en diferentes áreas, a ratos podríamos tener la concepción de que es una herramienta netamente informática, pero la verdad es que es aplicable a cualquier área del conocimiento y aprendizaje, como el desarrollo de una IA que maneje la información que imparte un profesor para poder consultar en cualquier minuto. Es bueno como se le da nuevas utilidades a diferentes áreas, como herramienta de ayuda y se le pierde este miedo un poco retrógrada de que la inteligencia nos quitara el "trabajo"

![image](https://github.com/community/community/assets/152074791/8be0b6d1-e4f8-431a-a14a-8a91877b4e28)

**Comment by michmelo** [comment]
Aprendiendo lo importante que puede ser una inteligencia artificial para el desarrollo de nuestras carreras pero sin hacer uso abusivo de esta 🤓


![IMG_20240409_171158](https://github.com/community/community/assets/166557963/29cb5261-c793-4648-ba68-45a5c71e9ed0)

**Comment by CrissDavid484** [comment]
Gracias a esto por fin pude tener la oportunidad de entender íntegramente la mejor forma de utilizar Copilot, fascinado de aprender y poder estar presente en esta charla. Me enamoré de todos los expositores, todos maravillosos en sus temas.
![17126973116976512932496122158270](https://github.com/community/community/assets/131937553/feff16bf-951f-4e66-a50c-bf7a9cba12d9)

**Comment by kike4466** [comment]
![Screenshot_2024-04-09-17-12-10-644_com google android youtube](https://github.com/community/community/assets/18172475/bcc7a3de-5690-445f-b603-950bb9d68c7c)

Es muy interesante ver una forma correcta de utilizar la iA, que ha muchos nos han dado a entender que nos quitaría el trabajo en el futuro, pero al ver el cómo lo utilizan realmente los profesionales, es de gran valor para quienes nos dedicaremos a la programación.

**Reply by kike4466** [reply]
![Screenshot_2024-04-09-17-17-25-502_com google android youtube](https://github.com/community/community/assets/18172475/bd5d37d0-0f43-452f-8b77-6b4f134cfcb6)

**Comment by Scottyneitor** [comment]
Buena oportunidad para aprender de Github, además como utilizar de una manera eficiente la manera eficiente la IA y lograr mejorar la eficiencia, tanto al momento de aprender y al momento de programar.
![Uploading 17126974405638077034076950816395.jpg…]()

**Comment by Be-Spring** [comment]
Alumna de primer año, enamorandome cada vez más de la carrera que elegí. Super importante iniciar con esta charla y tremendas oportunidades que nos dan. Muchas gracias a los expositores por su tiempo.
![20240409_170538](https://github.com/community/community/assets/166559788/58a80322-068f-4be5-9343-4d8ecaf4cb1d)

**Comment by Sebastian-p-c** [comment]
![IMG_20240409_162016](https://github.com/community/community/assets/134718272/6068e392-58e7-485c-b671-9d174330354b)
Increíble charla, pude aprender funciones que tiene Github Copilot que no sabía, y desde ahora en adelante lo ocuparé para poder tener mejor rendimiento personal, también me interesó el Open source el poder ayudar y aportar en proyectos de otras personas, muchas gracias por la charla.

**Comment by Darck-JS** [comment]
Una gran oportunidad para aprender las herramientas que otorga Github y Github Copilot, ya sea con apoyo de la IA y asi mejorar como programador, asi como también el ser parte de grandes futuros proyectos con muchas personas la cual de igual manera quiere ser parte de algo mucho mas grande, por lo que esto es siempre algo muy maravilloso de conocer y utilizar en el dia a día 
![Screenshot_20240409_171310](https://github.com/community/community/assets/164104117/73400962-9099-46f6-8e20-8ba415b7b0de)

**Comment by SimonKaiak** [comment]
Github Copilot me parece simplemente fantastico. El como influye la IA con lo que estas programando, como se relaciona con el cliente y se adapta lo que quiere conseguir. 
![Uploading 17126975004725964790055711145477.jpg…]()

**Comment by MadWorth** [comment]
Nunca habia visto una IA interactuar en tiempo real con uno mismo, un codigo y que lo haga tan bien, genial las demostraciones, se aprende muchísimo de esta manera.
![Uploading 20240409_171301.jpg…]()

**Comment by Bnjaaja** [comment]
que interesante los puntos que hablan, me llamo la atención como la IA, que es algo tan "nuevo", puede aportar tanto a algo de tan importante como es la enseñanza, el como puede potenciar tu conocimiento como una gran herramienta de aprendizaje y también como te puede afectar dependiendo del uso que tengas.
que bkn verla en funcionamiento, (10/10)
Conocia algo de github, pero ahora me doy cuenta que es bastante interesante.👍👍👍
![image](https://github.com/community/community/assets/166559487/90768449-8d27-404e-ba2b-13c5034481d5)
![github](https://github.com/community/community/assets/166559487/707bcb15-467a-46f7-8655-bf21d4fe1305)

**Comment by FG-270** [comment]
![20240409_171750](https://github.com/community/community/assets/166559600/33552820-f5e7-444d-9095-985fd7a26a95)
Muy buena charla. Me interesa mucho lo que son los programas open source, ya que me las he topado varias veces y me llaman la atención.

**Comment by MSFP5** [comment]
Me pareció interesante su analogía de como podemos utilizar IA para que sea un tutor y una herramienta de  apoyo para aprender, tambien, como las preguntas correctas pueden ampliar nuestro camino y nuestra mente.
Excelente charla!! 
![20240409_171742](https://github.com/community/community/assets/166559717/fa699b68-dd0a-428f-9834-a3f3d6825513)

**Comment by F-brizzio** [comment]
![Uploading image.jpg…]()
Aprendí mucho, muchas gracias

**Comment by Juanjomcf** [comment]
Es demasiado interesante la charla, sobre todo el tema de como se educa actualmente en las clases de programación y como el 1:1 da mejores resultados que la forma de 1 a todos. Copilot es una excelente herramienta también que se mencionó y no sabía cómo funcionaba del todo
![Uploading IMG_20240409_162532.jpg…]()

**Comment by deadhunter2115** [comment]
Aprendí sobre la distintas aplicaciones que le podemos dar  a la IA , a como contribuir en comunidad con proyectos open source, También implementar funciones , variables en copilot


![Imagen de WhatsApp 2024-04-09 a las 17 08 51_d88ac46f](https://github.com/community/community/assets/166559774/25d0a818-6282-43c6-b9f2-14b64edafa05)
![Imagen de WhatsApp 2024-04-09 a las 17 08 52_8d2de680](https://github.com/community/community/assets/166559774/1bee3e34-094d-47e5-910a-36dab691cab5)
![Imagen de WhatsApp 2024-04-09 a las 17 08 51_cc380e13](https://github.com/community/community/assets/166559774/26126c0b-93c4-41bb-b086-c44249d991c7)
![Imagen de WhatsApp 2024-04-09 a las 17 25 47_dd88cd9a](https://github.com/community/community/assets/166559774/ca3dc7b9-8d84-463b-a61d-ab25d51bccf7)

**Comment by DanielaD1az** [comment]
![17126980928968733830166254706791](https://github.com/community/community/assets/134526237/5aa5b322-3c61-4103-9ac8-02461f68bac0)
Feliz porque vine a aprender de GitHub con mis amigos y nos llevamos stickers.
Stikers y conocimientos, qué mejor!

**Comment by TomasDelFierro** [comment]
Hola!!!, primero gracias por la charla y les quería comentar que aprendí la importancia del correcto uso de la IA para desarrollarnos como profesionales en las TI.
Gracias. Saludos.

![320992415-ea33f503-36c5-4cdb-a265-c06f7b912e03](https://github.com/community/community/assets/166560620/61354c5a-229e-4797-8345-ed162d447c57)

**Comment by Skalibxr2** [comment]
![Screenshot_20240409_172547](https://github.com/community/community/assets/166560278/6e49f5c4-968a-47b8-8f4d-d46c52bfd884)

Con muchas ganas de aprender github para seguir creciendo en este mundo. Muchas gracias . Aprendí que no hay que abusar del uso de las IAs y usarlas más cómo un apoyo.
